recipes/libs/libsndfile.yaml

```diff
@@ -0,0 +1,27 @@
+inherit: [autotools, autoconf]
+
+metaEnvironment:
+  PKG_VERSION: "1.2.2"
+  PKG_LICENSE: "LGPL-2.1"
+
+depends: []
+
+checkoutSCM:
+  scm: url
+  url: ${GITHUB_MIRROR}/libsndfile/libsndfile/releases/download/${PKG_VERSION}/libsndfile-${PKG_VERSION}.tar.xz
+  digestSHA256: 3799ca9924d3125038880367bf1468e53a1b7e3686a934f098b7e1d286cdb80e
+  stripComponents: 1
+
+checkoutDeterministic: True
+checkoutScript: |
+  autoconfReconfigure
+
+buildTools: [python3]
```

Comment:
Is python just needed during the build (like make) or does the "installed" result have some python files? In case of the former, a `buildToolsWeak` is better because the exact python version does not matter.